src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.targets

```diff
@@ -0,0 +1,37 @@
+<!--
+***********************************************************************************************
+Microsoft.Bcl.Compression.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
```

Comment:
I find this header overly chatty, plus it doesn't make sense in a NuGet package since they aren't installed machine state. Plus the filename is wrong.